### DISCUSSION TITLE
LURO 5M - pending results game

### DISCUSSION THREAD

**Discussion by JackHipeek** [betfinio/public#59] [Uncalculated rounds]
On this game still showing pending status on result. 
<img width="794" alt="Snímek obrazovky 2024-10-07 v 23 36 35" src="https://github.com/user-attachments/assets/aa246d44-afaa-49aa-ad44-682c65527f5b">

**Comment by bf-monster** [comment]
Hey, should be calculated by now, please confirm

**Reply by JackHipeek** [reply]
nope. still same. Pending
<img width="813" alt="Snímek obrazovky 2024-10-09 v 23 16 11" src="https://github.com/user-attachments/assets/b96b6347-c711-46cc-97c3-12e6b4004946">

**Reply by bf-monster** [reply]
Round is calculated, but each bet has no updated its status. We will investigate

**Reply by bf-monster** [reply]
should be reseolved
